blogReader.py

```diff
@@ -16,18 +16,20 @@ def __init__(self) -> None:
         self.__new = []
         self.__getArticles()
         self.__cacheArticlesID()
-        self.handler = logging.getLogger(__name__)
+        self.logger = logging.getLogger(__name__)
 
     def __getArticles(self):
         if self.__cache == []:
             self.__cache = requests.get(self.url).json()
             return
         self.__new = requests.get(self.url).json()
+        self.logger.info("'getArticles' request sent")
     
     def __cacheArticlesID(self):
         self.__cachedArticlesID = []
         for dic in self.__cache:
             self.__cachedArticlesID.append(dic['_id'])
+        self.logger.info("'cacheArticlesID' updated")
```

Comment:
Actually, reading teh `__init__` where this function is called I think this is `initialized` not `updated` :wink:

Reply:
It seems that the recache function also calls __cacheArticlesID so that's why I thought it could also be an update.

Reply:
that's ok to me!